src/main/java/com/example/accountingapp/entity/StockDetails.java

```diff
@@ -0,0 +1,29 @@
+package com.example.accountingapp.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@NoArgsConstructor
```

Comment:
curious why you did not include is deleted.
I wonder if I should also remove it from my table:) 
I approve but maybe we can discuss when we meet

Reply:
Good catch!
This is the only entity without is_deleted in the Database Design (E-R Diagram)